.github/workflows/examples_test.yml

```diff
@@ -27,6 +27,8 @@ jobs:
         python examples/noisy_label_detection/trak_noisy_label.py --device cpu
         python examples/pretrained_benchmark/influence_function_lds.py --device cpu
         python examples/pretrained_benchmark/trak_loo.py --device cpu
+        sed -i 's/* 10/* 2/g' examples/pretrained_benchmark/trak_dropout_lds.py
```

Comment:
This line is using `sed` to change the file (only during the test) and reduce the computational cost (ensemble number changed from 10 to 2). It will not affect the raw file.

Reply:
got it